plugins/locale.js

```diff
@@ -1,30 +1,33 @@
 import { join } from "node:path";
-import { existsSync, readFileSync } from "node:fs";
+import { exists } from "node:fs/promises";
```

Comment:
exists does not exist for promises due to potential for race conditions.
You can use stat or access instead.

```js
fs.stat('foo.txt', function(err, stat) {
  if (err == null) {
    console.log('File exists');
  } else if (err.code === 'ENOENT') {
    // file does not exist
    fs.writeFile('log.txt', 'Some log\n');
  } else {
    console.log('Some other error: ', err.code);
  }
});
```

or maybe something like

```js
function checkFileExists(file) {
  return fs.promises.access(file, fs.constants.F_OK)
           .then(() => true)
           .catch(() => false)
}
```